src/FilterContext.ts

```diff
@@ -0,0 +1,22 @@
+// (C) 2019 GoodData Corporation
+
+export type FilterContextItem = IAttributeFilter | IDateFilter;
```

Comment:
These Types look like conflict with https://github.com/gooddata/gooddata-typings/blob/master/src/AFM.ts#L118

Reply:
I don't think so.
This type must be refelcted to backend [specification](https://github.com/gooddata/gdc-webapp/blob/develop/specification/md/obj.res#L1752)